README.md

```diff
@@ -158,10 +158,14 @@ Multiple sequence alignment file(s) (suchlike ARTIC Network multiple sequence al
 
 **-unaligned**: Specifies unaligned input files
 
-**-mask_problematic_sites**: Ignore problematic sites per masking recomendations
+**-auto_mask**: Ignore problematic sites per masking recomendations
+
+**-user_specified_mask**: Path to VCF fle containing custom masking recomendations (please ensure VCF format is consistent with https://raw.githubusercontent.com/W-L/ProblematicSites_SARS-CoV2/master/problematic_sites_sarsCov2.vcf)
 
 ### Usage
 
+Please ensure that MAFFT (see https://mafft.cbrc.jp/alignment/software/ ) is in your system path.
```

Comment:
I am currently not requiring the users to install any package on their own. All necessary installation should take place via the install scripts (installUbuntu.sh, installMacOS.sh and installCentOS.sh). Are you able to add the necessary instructions for mafft install in the three install scripts? You can assume sudo privileges. Docker script currently uses installUbuntu.sh, so that should automatically work as well. Thanks.

Reply:
yes will do rn